react/features/unsupported-browser/components/UnsupportedDesktopBrowser.js

```diff
@@ -40,45 +40,45 @@ class UnsupportedDesktopBrowser extends Component<Props> {
      */
```

Comment:
Wozu diese Änderungen in der Datei?

Reply:
Entspricht so dem Original mit Ausnahme des Textes:

https://github.com/jitsi/jitsi-meet/blob/master/react/features/unsupported-browser/components/UnsupportedDesktopBrowser.js

Reply:
Das ist bis auf den geänderten Text so im Original drin:

https://github.com/jitsi/jitsi-meet/blob/master/react/features/unsupported-browser/components/UnsupportedDesktopBrowser.js

Reply:
Wir wollen aber nicht den Master Mergen